src/graphics/bargraphics.jl

```diff
@@ -50,16 +52,23 @@ end
 
 function printrow(io::IO, g::BarplotGraphics, row::Int)
     0 < row <= nrows(g) || throw(ArgumentError("Argument \"row\" out of bounds: $row"))
+    nsyms = length(g.symbols)
     bar = g.bars[row]
     max_freq = g.max_freq
     max_bar_width = max(g.char_width - 2 - g.max_len, 1)
     val = g.transform(bar)
-    bar_len = max_freq > 0 ? round(Int, max(val, zero(val))/max_freq * max_bar_width, RoundNearestTiesUp) : 0
-    bar_str = max_freq > 0 ? repeat(g.symb, bar_len) : ""
+    frac = max_freq > 0 ? max(val, zero(val)) / max_freq : 0
+    bar_head = round(Int, frac * max_bar_width, RoundDown)
+    print_color(g.color, io, max_freq > 0 ? repeat(g.symbols[nsyms], bar_head) : "")
+    if nsyms > 1 && frac > 0
+        syms_tail = g.symbols[1:(nsyms-1)]
+        rem = (frac - bar_head / max_bar_width) / frac * (length(syms_tail) - 1)
```

Comment:
`rem = (frac*max_bar_width - bar_head) * (length(syms_tail) - 1)` seems to give more realistic bars

```julia
julia> using UnicodePlots, StatsBase

julia> UnicodePlots.histogram(Histogram(1.0:8,[0,2,4,6,8,10,400])) # before
              ┌                                        ┐
   [1.0, 2.0) ┤ 0
   [2.0, 3.0) ┤▉ 2
   [3.0, 4.0) ┤▉ 4
   [4.0, 5.0) ┤▉ 6
   [5.0, 6.0) ┤▉ 8
   [6.0, 7.0) ┤▉ 10
   [7.0, 8.0) ┤███████████████████████████████████▏ 400
              └                                        ┘
                              Frequency

julia> UnicodePlots.histogram(Histogram(1.0:8,[0,2,4,6,8,10,400])) # after
              ┌                                        ┐
   [1.0, 2.0) ┤ 0
   [2.0, 3.0) ┤▎ 2
   [3.0, 4.0) ┤▍ 4
   [4.0, 5.0) ┤▌ 6
   [5.0, 6.0) ┤▋ 8
   [6.0, 7.0) ┤▊ 10
   [7.0, 8.0) ┤███████████████████████████████████▏ 400
              └                                        ┘
```